services/nomad/build/buildbot-builder.nomad

```diff
@@ -0,0 +1,125 @@
+job "buildbot-builder" {
+  type = "service"
+  // TODO
+  // datacenters = ["VOID"]
+  // namespace = "build"
+  datacenters = ["dc1"]
+
+  dynamic "group" {
+    for_each = [
+      // TODO: jobs = 16, 6, 3
+      { name = "glibc",   jobs = 2,  admin = { nick = "maldridge", name = "maldridge",       mail = "maldridge@voidlinux.org" } },
+      { name = "musl",    jobs = 2,  admin = { nick = "maldridge", name = "maldridge",       mail = "maldridge@voidlinux.org" } },
+      { name = "aarch64", jobs = 2,  admin = { nick = "leah2",     name = "Leah Neukirchen", mail = "leah@vuxu.org" } }
+    ]
+    labels = [ "buildbot-builder-${group.value.name}" ]
+
+    content {
+      dynamic "volume" {
+        for_each = [ "${group.value.name}" ]
+        labels = [ "${volume.value}_hostdir" ]
+
+        content {
+          type = "host"
+          source = "${volume.value}_hostdir"
+          read_only = false
+        }
+      }
+
+      // https://github.com/hashicorp/nomad/issues/8892
+      task "prep-hostdir" {
+        driver = "docker"
+
+        config {
+          image = "ghcr.io/void-linux/void-glibc:latest"
+          command = "sh"
+          args = ["-c", "chown -R 5000:5000 /hostdir"]
+        }
+
+        volume_mount {
+          volume = "${group.value.name}_hostdir"
+          destination = "/hostdir"
+        }
+
+        lifecycle {
+          hook = "prestart"
+        }
+      }
+
+      task "buildbot-worker" {
+        driver = "docker"
+
+        user = "void-builder"
+
+        config {
+          image = "localhost:5000/infra-buildbot-builder:latest"
```

Comment:
This should be versioned, and should be kept in step with the prep container above to optimize cache usage.